pkg/controller/knativeeventingkafka/knativeeventingkafka_controller.go

```diff
@@ -40,12 +34,16 @@ var (
 // Add creates a new KnativeEventingKafka Controller and adds it to the Manager. The Manager will set fields on the Controller
 // and Start it when the Manager is Started.
 func Add(mgr manager.Manager) error {
-	return add(mgr, newReconciler(mgr))
+	manifest, err := mf.NewManifest(*filename, *recursive, mgr.GetClient())
+	if err != nil {
+		return err
+	}
+	return add(mgr, newReconciler(mgr, manifest))
 }
 
 // newReconciler returns a new reconcile.Reconciler
-func newReconciler(mgr manager.Manager) reconcile.Reconciler {
-	return &ReconcileKnativeEventingKafka{client: mgr.GetClient(), scheme: mgr.GetScheme()}
+func newReconciler(mgr manager.Manager, man mf.Manifest) reconcile.Reconciler {
+	return &ReconcileKnativeEventingKafka{client: mgr.GetClient(), scheme: mgr.GetScheme(), config: man}
```

Comment:
I prefer the `InjectClient` than restoring these. It keeps things more like the osdk scaffolding, which makes updates easier.